plugins/repository-hdfs/src/main/java/org/elasticsearch/repositories/hdfs/HdfsSecurityContext.java

```diff
@@ -67,10 +67,13 @@
             new AuthPermission("modifyPrincipals"),
             new PrivateCredentialPermission("org.apache.hadoop.security.Credentials * \"*\"", "read"),
             new PrivateCredentialPermission("javax.security.auth.kerberos.KerberosTicket * \"*\"", "read"),
-            new PrivateCredentialPermission("javax.security.auth.kerberos.KeyTab * \"*\"", "read")
+            new PrivateCredentialPermission("javax.security.auth.kerberos.KeyTab * \"*\"", "read"),
             // Included later:
             // 7) allow code to initiate kerberos connections as the logged in user
             // Still far and away fewer permissions than the original full plugin policy
+            // The following are required by CryptoOutputStream:
+            new RuntimePermission("accessClassInPackage.sun.nio.ch"),
+            new RuntimePermission("accessClassInPackage.sun.misc")
```

Comment:
Nit: Can we order these above the `// Included later` line and make sure the ordering of these matches up with the ordering with the policy file?